src/Storefront/Resources/app/storefront/src/plugin/captcha/google-re-captcha/google-re-captcha-v2.plugin.js

```diff
@@ -30,6 +30,7 @@ export default class GoogleReCaptchaV2Plugin extends GoogleReCaptchaBasePlugin
     }
 
     onFormSubmit() {
+        debugger;
```

Comment:
:rotating_light:

Reply:
Was curious: Looks like ESLInt didn't complain about this 😢 , would be another opportunity for a good static linting rule 😅 
https://github.com/shopware/shopware/actions/runs/15486751062/job/43603061693?pr=10337